src/components/AudioTrack/layouts/Row.vue

```diff
@@ -82,6 +84,11 @@ export default {
       isSmall,
     }
   },
+  data() {
+    return {
+      seperatorCharacter: '•',
+    }
+  },
```

Comment:
Since component uses setup, we can export the data from there itself.

Reply:
😊 Great suggestion!